### PR TITLE
[PKG-6375] Helm docs (standard and includes OCI based)

### PR DESCRIPTION
**Pull request by tommeier**
Extension of https://github.com/buildkite/docs/pull/2926 (and built on top of the branch)

CleanShot 2024-08-19 at 09 41 56@2x

CleanShot 2024-08-19 at 09 42 26@2x

Note: I opted to use "OCI-based" as the differentiator given this is the wording used on [helm docs ](https://helm.sh/docs/topics/registries/)

CleanShot 2024-08-19 at 09 43 18@2x

### CODE REVIEW

**Review by pinzonjulian — APPROVED** [review]

File: pages/packages/helm.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+To view and copy this `curl` command:
+
+1. Select **Packages** in the global navigation to access the **Registries** page.
+1. Select your Helm registry on this page.
+1. Select **Publish a Helm Chart** and in the resulting dialog, use the copy icon at the top-right of the code box to copy this `curl` command and run it to publish your chart to your Helm registry.
+
+This command provides:
+
+- The specific URL to publish a package to your specific Helm registry in Buildkite.
+- The API access token required to publish packages to your Helm registry (if private registry).
```
Comment (isaacsu) [inline pages/packages/helm.md:18]:
```suggestion
- The API access token required to publish packages to your Helm registry.
```

Unless no access token is needed to publish to a public Helm registry?

File: pages/packages/_helm_registry_slug.md [hunk 2]
```diff
@@ -0,0 +1 @@
+- `{registry.slug}` is the slug of your Helm registry, which is the [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) version of your Helm registry name, and can be obtained after accessing **Packages** in the global navigation > your helm registry from the **Registries** page.
```
Comment (isaacsu) [inline pages/packages/_helm_registry_slug.md:1]:
```suggestion
- `{registry.slug}` is the slug of your Helm registry, which is the [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) version of your Helm registry name, and can be obtained after accessing **Packages** in the global navigation > your Helm registry from the **Registries** page.
```

File: pages/packages/helm.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
+# Helm
+
+Buildkite Packages provides Helm registry support for distributing Helm charts.
+
+This page is for standard helm publishing instructions, alternatively you can also publish to an [OCI-based registry](/docs/packages/helm-oci).
```
Comment (isaacsu) [inline pages/packages/helm.md:5]:
```suggestion
This page is for standard Helm publishing instructions, alternatively you can also publish to an [OCI-based registry](/docs/packages/helm-oci).
```

File: pages/packages/helm.md (hunk 1 above)
Comment (tommeier) [reply pages/packages/helm.md:18]: heh good catch

**Review by isaacsu — APPROVED** [review]

**Comment by tommeier** [comment]
Helm OCI & Standard is now live in production - will merge this now, but as always @gilesgas @mbelton-buildkite please update/change anything and everything 🙏

**Pull request merged**